index.html

```diff
@@ -21,237 +21,218 @@ <h1>The open source multicloud control plane.</h1>
       </div>
-      <div class="col-6_md-12_md-first img">
-        <img class="md-hidden" src="{{ "/images/feature-2.svg" | relative_url }}" /><img
-          class="md-gt-hidden"
-          src="{{ "/images/feature-2-mobile.svg" | relative_url }}"
-        />
+      <div class="blueprint-item col-11">
+        <h3>Full lifecycle management of resources.</h3>
+        <p>A resource controller is responsible for the entire lifecycle of a resource. This 
+          resource is responsible for provisioning, health, scaling, failover, and actively 
+          responding to external changes that deviate from the desired configuration.</p>
       </div>
     </div>
-    <div class="grid">
-      <div class="col-6_md-12 img">
-        <img class="md-hidden" src="{{ "/images/feature-3.svg" | relative_url }}" /><img
-          class="md-gt-hidden"
-          src="{{ "/images/feature-3-mobile.svg" | relative_url }}"
-        />
-      </div>
-      <div class="col-6_md-12">
-        <h2>Automate all of your tasks across multiple clouds.</h2>
-        <p>
-          Write custom controllers to automate such tasks as setting up
-          replication across databases, or setting up a data protection solution
-          across clouds. With a single control-plane that can span regions and
-          cloud providers, and offers an extensibility model for writing custom
-          controllers, Crossplane automates many of these tasks.
-          <img src="{{ "/images/quad-white-medium.svg" | relative_url }}" />
-        </p>
-      </div>
+    <div class="blueprint-img col-6_md-hidden">
+      <img src="{{ "/images/blueprint.png" | relative_url }}" alt="open cloud blueprint" />
     </div>
   </div>
 </div>
+
 <div id="testimonials">
   <div>
     <div class="grid-center">
-      <div class="testimonial col-4_md-6_sm-12">
+      <div class="testimonial col-6_md-12">
         <h3>
           &ldquo;Crossplane has an opportunity to change the cloud industry as
-          we know it&rdquo;
+          we know it.&rdquo;
         </h3>
         <p>
           &ldquo;Our customers are increasingly looking for a way to deploy
           their code across multiple cloud environments. The choices available
-          today are too complex and vendor driven but with Crossplane the
+          today are too complex and vendor-driven, but with Crossplane, the
           ability to orchestrate clouds becomes simple. We look forward to
           collaborating with them on this vision and as the first complex app
           running on Crossplane.&rdquo;
         </p>
         <div class="signature">
-          <div class="name">Sid Sijbrandij</div>
-          <div class="title">CEO and Co-Founder of GitLab</div>
-          <a href="https://gitlab.com"><img src="{{ "/images/gitlab.svg" | relative_url }}"/></a>
+          <div class="exec-img">
+            <img srcset="./images/sid.jpg 1x, ./images/sid@2x.jpg 2x, ./images/sid@3x.jpg 3x" alt="Sid Sijbrandij" />
```

Comment:
I believe you should still have a `src` along with the `srcset` and all of these should use the `relative_url` helper